frontend/src/app/shared/components/dashboard/tweet-card/tweet-card.component.html

```diff
@@ -1 +1,29 @@
-<p>tweet-card works!</p>
+<div class="card d-flex rounded">
+  <div class="d-flex justify-content-between">
+    <div class="username m-3">
+      <img
+        class="avatars"
+        src="https://i.imgur.com/73hZDYK.png"
+        alt="User Avatar"
+      />
+    </div>
+    <div class="m-3">
+      <span>Newton</span>
```

Comment:
- [ ] if no special Styling is applied - no need to wrap text to <span> element. Remove it.